core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/transfer/TransferProcessManagerImpl.java

```diff
@@ -52,6 +52,7 @@
 import org.eclipse.dataspaceconnector.spi.types.domain.transfer.TransferProcessStates;
 import org.eclipse.dataspaceconnector.spi.types.domain.transfer.command.TransferProcessCommand;
 
+import java.time.Clock;
```

Comment:
Yes, this is used in test and is standard best practice. Any reason not to?

One improvement down the line could be to make the clock global.

Reply:
if we decide to use `clock` instead of `Instant.now()....` we should do it in one fell swoop across the code base in a dedicated PR, not piece-by-piece.